### PR TITLE
Adding vale spell and style checking

### PR DESCRIPTION
**Pull request by spier**
Adding basic setup for vale (spell and style checking)

### CODE REVIEW

**Comment by spier** [comment]
Looks like the checks are working, as the create various warnings here:
https://github.com/InnerSourceCommons/managing-inner-source-projects/pull/46/checks?check_run_id=17923812175

Note the warnings are all filtered out, as I did not touch any of the markdown files in this PR.

**Comment by spier** [comment]
Made a couple of minor spelling fixes myself.

There are just [12 reported issues ](https://github.com/InnerSourceCommons/managing-inner-source-projects/actions/runs/6597312775/job/17924229806)remaining now, which I think we can best work out when we start touching the content again anyways.

Marking this PR as ready for review now :)

**Review by rrrutledge — APPROVED** [review]
This is wonderful!  Thank you!

File: .github/vale/Vocab/Base/accept.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+toolchain
+archivable
+
+# people
+Jono
+Bacon
+Karl
+Fogel
+
+# Orgs
```
Comment (spier) [inline .github/vale/Vocab/Base/accept.txt:12]:
```suggestion
# orgs
```

**Pull request merged**